proto/redpanda/api/dataplane/v1alpha1/topic.proto

```diff
@@ -0,0 +1,261 @@
+syntax = "proto3";
+
+package redpanda.api.dataplane.v1alpha1;
+
+import "google/api/annotations.proto";
+import "google/api/field_behavior.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
+
+message Topic {
+  message Configuration {
+    enum Source {
+      SOURCE_UNSPECIFIED = 0;
+      SOURCE_DYNAMIC_TOPIC_CONFIG = 1;
+      SOURCE_DYNAMIC_BROKER_CONFIG = 2;
+      SOURCE_DYNAMIC_DEFAULT_BROKER_CONFIG = 3;
+      SOURCE_STATIC_BROKER_CONFIG = 4;
+      SOURCE_DEFAULT_CONFIG = 5;
+      SOURCE_DYNAMIC_BROKER_LOGGER_CONFIG = 6;
+    }
+
+    string name = 1;
+    string type = 2;
+    string value = 3;
+    Source source = 4; // Enum ?
+    bool is_read_only = 5;
+    bool is_sensitive = 6;
+
+    message ConfigSynonym {
+      string name = 1;
+      string value = 2;
+      Configuration.Source source = 3; // Enum?
```

Comment:
Not sure the `Enum?` comment makes sense?

Reply:
Same as above, the protos for anything but user.proto are just drafts for the OpenAPI spec.